eng/azure-pipelines.yml

```diff
@@ -4,6 +4,10 @@ trigger:
   paths:
     exclude: [".github", "Documentation", "*.md"]
 
+variables:
+  system.debug: false
+  EnableCoverage: true
```

Comment:
There is an issue with the `/p:CollectCoverage=true` parameter which activates coverlet.msbuild code coverage.This is kind of a blocker for net8.0 transition because a lock will stop DLL copy operation.

- coverlet.core.test
- coverlet.collector.test

are using
`<Import Project="$(RepoRoot)src\coverlet.msbuild.tasks\coverlet.msbuild.target />`

I will revert this and remove the test without enabled code coverage. This is not relevant for this PR.